N°7219 - Dashboard issue (fix 2.7)

bdalsass opened the pull request:
Problème d'encodage des dashboards.

https://support.combodo.com/pages/UI.php?operation=details&class=Bug&id=7219&

bdalsass committed:
N°7219 - Fatal error following dashboard modification when dashboard title contains an é

rquetiez commented on application/dashboard.class.inc.php:
```diff
@@ -1197,8 +1197,8 @@ public function RenderEditor($oPage, $aExtraParams = array())
 		$sAutoReloadSec = (string) $this->iAutoReloadSec;
-		$sTitle = utils::HtmlEntities($this->sTitle);
-		$sFile = utils::HtmlEntities($this->GetDefinitionFile());
+		$sTitle = utils::EscapeHtml($this->sTitle);
```
Un double encodage va persister sur les caractères comme &, < et >. De plus, je pense que cela va régresser si le titre contient une simple quote. json_encode() serait l'outil approprié pour ce cas.

bdalsass committed:
N°7219 - Fatal error following dashboard modification when dashboard title contains an é
- Remplacement des appels à utils::EscapeHtml et utils::HtmlEntities par json_encode et généralisation au scope de la fonction.
